styleguide/cypress/integration/styleguide-spec.js

```diff
@@ -1,5 +1,8 @@
 describe('styleguide', () => {
   it('works', () => {
+    cy.on('uncaught:exception', () => {
```

Comment:
Does this fail the build on uncaught exceptions? It might be nice to include an error message in that case.

Reply:
There's still 1 minor incompatibility with React Styleguidist and Webpack 5. It errors within the websocket connection for hot module reloading. 

I made an issue, we'll see if someone has better context on how to fix it in upstream.